backend/ethereum/channel/conclude.go

```diff
@@ -98,11 +128,10 @@ func (a *Adjudicator) ensureConcluded(ctx context.Context, req channel.Adjudicat
 // the Concluded event subscription instance.
 func waitConcludedForNBlocks(ctx context.Context,
 	cr ethereum.ChainReader,
-	sub ethereum.Subscription,
-	concluded chan *adjudicator.AdjudicatorChannelUpdate,
+	concluded chan *subscription.Event,
 	numBlocks int,
 ) (bool, error) {
-	h := make(chan *types.Header)
+	h := make(chan *types.Header, 10)
```

Comment:
Why with buffer now?

Reply:
I do not want to block geth. Using un-buffered channels has no advantage here IMHO.

Reply:
OK